### ISSUE TITLE
Node statistics not available

### ISSUE THREAD

**Issue by brandond** [kubernetes-sigs/dashboard-metrics-scraper#7]
I'm running EKS with metrics-server v0.3.2 and kubernetesdashboarddev/dashboard-metrics-sidecar:latest pulled today. Pod-level cpu and memory charts appear in the UI, but I don't see any node-level charts. I also see errors regarding node stats in the log.

```
{"level":"info","msg":"URL: /","time":"2019-04-22T20:33:57Z"}
{"level":"error","msg":"Error updating database: no such table: nodes","time":"2019-04-22T20:33:59Z"}
{"level":"info","msg":"URL: /","time":"2019-04-22T20:34:07Z"}
{"level":"info","msg":"Database updated","time":"2019-04-22T20:34:14Z"}
{"level":"info","msg":"URL: /","time":"2019-04-22T20:34:17Z"}
{"level":"info","msg":"Query: select sum(cpu), name, uid, time from nodes  where  name = ? group by name, time order by name, time;","time":"2019-04-22T20:34:18Z"}
{"level":"info","msg":"Values: [ip-10-12-135-107.us-west-2.compute.internal]","time":"2019-04-22T20:34:18Z"}
{"level":"info","msg":"Query: select sum(cpu), name, uid, time from nodes  where  name = ? group by name, time order by name, time;","time":"2019-04-22T20:34:18Z"}
{"level":"info","msg":"Values: [ip-10-12-134-62.us-west-2.compute.internal]","time":"2019-04-22T20:34:18Z"}
{"level":"info","msg":"Query: select sum(cpu), name, uid, time from nodes  where  name = ? group by name, time order by name, time;","time":"2019-04-22T20:34:18Z"}
{"level":"info","msg":"Values: [ip-10-12-133-167.us-west-2.compute.internal]","time":"2019-04-22T20:34:18Z"}
{"level":"info","msg":"Query: select sum(cpu), name, uid, time from nodes  where  name = ? group by name, time order by name, time;","time":"2019-04-22T20:34:18Z"}
{"level":"info","msg":"Values: [ip-10-12-133-94.us-west-2.compute.internal]","time":"2019-04-22T20:34:18Z"}
{"level":"info","msg":"Query: select sum(cpu), name, uid, time from nodes  where  name = ? group by name, time order by name, time;","time":"2019-04-22T20:34:18Z"}
{"level":"info","msg":"Values: [ip-10-12-134-207.us-west-2.compute.internal]","time":"2019-04-22T20:34:18Z"}
{"level":"info","msg":"Query: select sum(cpu), name, uid, time from nodes  where  name = ? group by name, time order by name, time;","time":"2019-04-22T20:34:18Z"}
{"level":"info","msg":"Values: [ip-10-12-132-112.us-west-2.compute.internal]","time":"2019-04-22T20:34:18Z"}
{"level":"error","msg":"Error getting node metrics: no such table: nodes","time":"2019-04-22T20:34:18Z"}
{"level":"error","msg":"Error getting node metrics: no such table: nodes","time":"2019-04-22T20:34:18Z"}
```

**Comment by brandond** [comment]
I haven't been able to reproduce this on demand. I set up the container with a writable temp volume and stuck the DB on there and the errors have gone away, but that could be circumstantial. I'll try to hack on it some more.

**Comment by maciaszczykm** [comment]
cc @jeefy

**Comment by jeefy** [comment]
Two things need addressing.

1) I also saw bizarre behavior with using a `:memory` sqlite database, and cannot explain why. Once it's hitting a file, it's stable. Because of that, I've changed the default database to be stored in `/tmp/metrics.db` This is fixed with #11 

2) Node-level metrics are stored by the metrics-scraper, however, Dashboard (2.0) does not display any charts for node metrics, only pods. I'll create an issue under k/dashboard to track this effort.
